recipe/meta.yaml

```
@@ -22,30 +21,38 @@ requirements:
     - {{ compiler('fortran') }}
     - cmake
     - make
+    - pkg-config
   host:
-    - libblas
-    - libcblas
-    - liblapack
+    - openblas  # [win]
+    - libblas  # [not win]
+    - libcblas  # [not win]
+    - liblapack  # [not win]
```

```suggestion
    - libblas
    - blas-devel
    - libcblas
    - liblapack
```

No need to hardcode a blas implementation, it is just needed to add `blas-devel` that is required for CMake to find mkl, that is the default blas on Windows (see https://github.com/conda-forge/conda-forge.github.io/pull/2171).